sass/navigation.scss

```diff
@@ -27,11 +42,10 @@
 
       h4 {
         font-family: $base-font;
-        font-size: 12px;
-        text-transform: uppercase;
+        font-size: 16px;
 
         a {
-          color: $orange;
+          color: $darkblue;
```

Comment:
It looks OK, but we do use orange for links everywhere else on the site. How bad does it it look if we use orange here too?

If we do use darkblue here, we should probably use the same colour for the little arrow on the active item in the sidebar.